index.html

```diff
@@ -230,11 +230,25 @@ <h2>Scope</h2>
 								new media types, such as OPUS, BPG, or WebP;
 							</li>
 							<li>
-								allowing the HTML Serialization of HTML5, including whether and how to handle xml-specific technologies like <code>epub:type</code> and the <code>ssml:*</code> attributes;
+								allowing the HTML Serialization of HTML5, including whether and how to handle xml-specific technologies like <code>epub:type</code> and the <code>ssml:*</code> attributes.
 							</li>
 						</ul>
 						Such new technical features should be incubated by the <a href="https://www.w3.org/community/publishingcg/">Publishing Community Group</a> before being considered by this Working Group.  Also, the effects on deployed systems and checking tools, and the risk of additional cost for adoption, must be carefully analyzed. In case of doubt, such features should not be added to EPUB 3.X.
 					</li>
+					<li>
+						Improvements of the <a href='#a11y'>EPUB Accessibility</a> specification; these may include, but not limited to:
+						<ul>
+							<li>
+								verify that EPUB Accessibility 1.X meets the <a href='https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=uriserv:OJ.L_.2019.151.01.0070.01.ENG&toc=OJ:L:2019:151:TOC'>EU Directive for accessibility requirements for eBooks</a> (European Accessibility Act, Directive 2019/882 of the European Parliament); the WG will conduct a review of the <a href='#a11y'>EPUB Accessibility</a> specification for meeting the EU requirements, provide clarifications, and add the missing pieces as required;
+							</li>
```

Comment:
It is better to say "and add the missing pieces
if required". We have already reviewed the EPUB Accessibility w.r.t EU directive. It already meets the requirements and mainly requires clarifications and adjustments to make it clear that all requirements are met. So, stating "as required" instead of "if required" would be over statement. We are open to adding  more features if we discover that something important is missing.

Reply:
Thanks @avneeshsingh, the commit https://github.com/w3c/epub-3-wg-charter/pull/47/commits/6f385697ac5e147a8d022f0d63c28345acd65520 takes care of those changes.